docs-md/developer-guide/ksqldb-reference/insert-values.md

```diff
@@ -14,13 +14,13 @@ Synopsis
 
 ```sql
 INSERT INTO <stream_name|table_name> [(column_name [, ...]])]
-  VALUES (value [,...]);
+  VALUES (value [,...])[, (value [,...])];
 ```
 
 Description
 -----------
 
-Produce a row into an existing stream or table and its underlying topic
+Produce rows into an existing stream or table and its underlying topic

```

Comment:
might be worth explicitly calling out that if one row doesn't match (i.e. has too many/not enough values) then none are produced. Is that the same behavior as other DBs?

Reply:
Yes, the behavior is the same  as other DBs. 

One thing to notice, though, is that if a row fails during the `Producer.sendRecord`, then all the previous successful rows will be produced. This is due to the non-transactional behavior we have. Now I wonder, is this a valid behavior we want? I cannot verify with other DBs if this is valid because I would need to mock the filesystem or something and fake one rows couldn't be written, but being other DBs transactional, I believe none of them will be committed.

Reply:
I think that behavior is OK - we can change it later if people are bothered by it